code/modules/antagonists/blob/blobstrains/_reagent.dm

```diff
@@ -27,7 +27,8 @@
 	can_synth = FALSE
 
 
-/datum/reagent/blob/expose_mob(mob/living/M, methods=TOUCH, reac_volume, show_message, touch_protection, mob/camera/blob/O)
-	if(M.stat == DEAD || istype(M, /mob/living/simple_animal/hostile/blob))
+/datum/reagent/blob/expose_mob(mob/living/exposed_mob, methods=TOUCH, reac_volume, show_message, touch_protection, mob/camera/blob/overmind)
+	SEND_SIGNAL(src, COMSIG_REAGENT_EXPOSE_MOB, exposed_mob, methods, reac_volume, show_message, touch_protection, overmind)
```

Comment:
Signals can't have multiple sources. This is a bit of a mess, redefining the proc and not calling parent.
I don't want to creep the scope of this, but this needs to be refactored.

Reply:
Yeah, it is a massive mess. I'll see what I can do.

Reply:
I think I'll need to atomize the fix for this out. I'll see what I can do.

Reply:
Never mind, using a set of bitflags. Reagent traits can be done another day.